File: GitToolBox/src/main/java/zielu/gittoolbox/completion/CurrentBranchCompletionContributor.java
package zielu.gittoolbox.completion;

import com.intellij.codeInsight.completion.CompletionContributor;
import com.intellij.codeInsight.completion.CompletionType;
import com.intellij.patterns.PlatformPatterns;
import com.intellij.psi.PsiPlainText;

public class CurrentBranchCompletionContributor extends CompletionContributor {
  public CurrentBranchCompletionContributor() {
    extend(CompletionType.BASIC, PlatformPatterns.psiElement(PsiPlainText.class),
        new CurrentBranchCompletionProvider());
  }
}

File: src/main/kotlin/zielu/gittoolbox/completion/CurrentBranchCompletionContributor.kt
package zielu.gittoolbox.completion

import com.intellij.codeInsight.completion.CompletionContributor
import com.intellij.codeInsight.completion.CompletionType
import com.intellij.patterns.PlatformPatterns
import com.intellij.psi.PsiPlainText

internal class CurrentBranchCompletionContributor() : CompletionContributor() {
  init {
    extend(
      CompletionType.BASIC, PlatformPatterns.psiElement(PsiPlainText::class.java),
      CurrentBranchCompletionProvider()
    )
  }
}
